src/Uno.UWP/Extensions/CoreDispatcherExtensions.cs

```diff
@@ -13,10 +13,9 @@ internal static class CoreDispatcherExtensions
 		/// </summary>
 		/// <typeparam name="T">The result type</typeparam>
 		/// <param name="dispatcher">The <see cref="CoreDispatcher"/> to run the task on</param>
-		/// <param name="priority">The task execution priority</param>
 		/// <param name="task">A function providing the task to execute</param>
 		/// <returns>The result of the task's execution</returns>
-		internal static async Task<T> RunWithResultAsync<T>(this CoreDispatcher dispatcher, CoreDispatcherPriority priority, Func<Task<T>> task)
```

Comment:
it feels like this should be the other way around :) The parameter was not used inside and should be.

Reply:
@Youssef1313 could you adjust this? Thanks!